redis/src/aio/connection.rs

```diff
@@ -24,6 +25,7 @@ use futures_util::{
     future::FutureExt,
     stream::{Stream, StreamExt},
 };
+use log::trace;
```

Comment:
please add a feature flag over this, I prefer not to add another dependency to `aio`.

Reply:
I prefer not to add an explicit feature to the crate - we have too many of them. I think that you can use `#[cfg(feature = "log")]` to check if the `log` crate is added as a feature.